src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -292,7 +292,10 @@ export class SavedSearchEmbeddable
     const timeRangeSearchSource = searchSource.create();
     timeRangeSearchSource.setField('filter', () => {
       if (!this.searchProps || !this.input.timeRange) return;
-      return this.services.timefilter.createFilter(indexPattern, this.input.timeRange);
+      return this.services.timefilter.createFilter(indexPattern, {
+        timeRange: this.input.timeRange,
+        coerceRelativeTimeToAbsoluteTime: false,
```

Comment:
i am wondering if we really need to change it in all this places ? at least embeddable i think should still do the conversion (for the sake of dashboards)

Reply:
I agree with @ppsiljar here